src/modules/business/dtos/IBusinessProductDTO.ts

```diff
@@ -0,0 +1,7 @@
+export default interface IBusinessProductDTO {
+  name: string;
+  description: string;
+  price: number;
+  url: string;
```

Comment:
change the name of this variable to make it more descriptive
```suggestion
  photo_product_url: string;
```